CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 * Add support for Python 3.9
 
+### Changed
+* Update `b2 sync` usage text for bucket-to-bucket sync
```

Comment:
did we change the requirements to make sure b2sdk is in the version that supports b2b sync?

Reply:
I believe it should be changed during the next release. Otherwise, we will break CI and you won't be able to install it locally (without tweaking requirements.txt).

E.g. SDK is now set to `1.1.5` (development version), but actually, according to SemVer, it should be released as 1.2.0 (we added new features, not only bug fixes). But if I set in CLI `b2sdk>=1.2.0,<1.4.0`, pip dails even if I pre-installed SDK from the master branch

Reply:
Then lets release sdk 1.2.0